### PR TITLE
Aliases for easier setup

### PR DESCRIPTION
**Pull request by filipecabaco**
Added multiple alias to facilitate startup and setup of deployment:
* start commands now source the .dev.env file
* added command to decrypt and encrypt dev .env file
* addec command to encrypt and decrypt staging .env and gcloud.json files

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -215,7 +223,19 @@ defmodule Logflare.Mixfile do
       "lint.diff": ["credo diff master"],
       "lint.all": ["credo --strict"],
       "ecto.setup": ["ecto.create", "ecto.migrate", "run priv/repo/seeds.exs"],
-      "ecto.reset": ["ecto.drop", "ecto.setup"]
+      "ecto.reset": ["ecto.drop", "ecto.setup"],
+      "decrypt.dev":
+        "cmd gcloud kms decrypt --ciphertext-file='./.dev.env.enc' --plaintext-file=./.dev.env --location=us-central1 --keyring=logflare-keyring-us-central1 --key=logflare-secrets-key --project=logflare-staging",
```
Comment (Ziinc) [inline mix.exs:228]: aliases looks good, seems like we missed out prod.
Could you backlog a task for moving these to a Mix.Task? we can probably reduce a lot of repetition.

**Review by Ziinc — APPROVED** [review]

File: mix.exs (hunk 1 above)
Comment (filipecabaco) [reply mix.exs:228]: didn't add production yet since we don't have that env set

**Comment by Ziinc** [comment]
@filipecabaco for CI, looks like docker push is getting triggered even though we haven't merged it yet into staging.

**Comment by filipecabaco** [comment]
@Ziinc it triggers the build to check it actually builds but doesn't push the image

**Pull request merged**